webstruct/webannotator.py

```diff
@@ -264,6 +186,107 @@ def _set_base(tree, baseurl):
     head = _ensure_head(tree)
     head.insert(0, Element("base", href=baseurl))
 
+TagPosition = namedtuple('TagPosition', ['element',
+                                         'tag',
+                                         'position',
+                                         'length',
+                                         'is_tail',
+                                         'dfs_number'])
```

Comment:
I think "dfs_number" implies something like "amount of 'dfs'" or "count of 'dfs'", but likely this is not what is meant here - is it an index/position of some kind?

Reply:
When we talk about depth first search it is common to call these indices numbers. [Wiki article](https://en.wikipedia.org/wiki/Depth-first_search#DFS_ordering) references whole process as enumeration.

Reply:
@whalebot-helmsman ok! Well, if you search "number" in this article, it won't have the meaning you're using here - it is only used as "number of nodes", "number of edges". But I can find "dfs number" mentioned in other articles, so that's fair to keep this name, you're right.